projects/core/abstract/driver.ts

```diff
@@ -1,6 +1,7 @@
 import {ExistingProvider, Type} from '@angular/core';
 import {Observable} from 'rxjs';
 
+// eslint-disable-next-line @typescript-eslint/naming-convention
 export abstract class TuiDriver extends Observable<boolean> {}
```

Comment:
@waterplea do we need rename to `AbstractTuiDriver`?

Reply:
I don't think so. It's kind of like Angular has some abstract classes that don't look like abstract in their name. I feel it is more natural this way for those.